### ISSUE TITLE
Titleception

### ISSUE THREAD

**Issue by jamalex** [learningequality/kolibri#1838]
## Summary

The new carousel and other UI improvements are so pretty, which is probably what made this pop out a bit more:

![image](https://user-images.githubusercontent.com/618416/28128771-c0581a4c-66e5-11e7-8cd6-6e501f812018.png)

Maybe no need for "Recommended" in the breadcrumb (so there wouldn't be a breadcrumb on the root page), and no need for the H1 on there or under Topics.

## System information

 - Version: 0.5.x

**Comment by jamalex** [comment]
On this note, do the breadcrumbs buy us anything at all under the Recommended tab?

![image](https://user-images.githubusercontent.com/618416/28128980-71c02e3c-66e6-11e7-8ab6-68324011c342.png)

1. It's only ever 2 levels deep
1. You can get back to the main page by clicking the "Recommended" tab again
1. The title is repeated right below that

**Comment by jtamiace** [comment]
^ I believe there is a redesign that will not require this breadcrumb.

<img width="1113" alt="screen shot 2017-07-12 at 9 43 05 am" src="https://user-images.githubusercontent.com/6668144/28129026-ad71dea8-66e6-11e7-814e-f790732eee64.png">

Also wouldn't be opposed to getting rid of the breadcrumb name (on first page only) and "Topics"

**Comment by jamalex** [comment]
> do the breadcrumbs buy us anything at all under the Recommended tab?

I know this has been discussed, but I still actually feel it could be useful to have the *topic-tree-based* breadcrumbs show up on content pages instead, under Recommended, as it's useful to see where you are in the tree even when you popped in from the Recommended tile page. E.g. I go to "resume" something that I previously navigated to via the topic tree, and then want to keep going through, but I don't have the context of where I am in the topic and no way to jump *to* the topic.

**Comment by jtamiace** [comment]
If the important thing is to see context, I would rather the breadcrumb be represented in a secondary way. Maybe some sort of "from <Topic>" link in the metadata section of the page to allow that jump to see the bigger context of that content, and from there, the learner could see the topic tree?

**Comment by jamalex** [comment]
Something like that could work as well, but consistency seems valuable for the user, so having it be the same layout and contextual structure as when visited via the topic tree could be good. I know we've gone back and forth on this previously, and I think part of the reason of keeping them separate was to have "Recommended" be more of a guided experience, but I don't know that we're currently achieving that anyway (especially until we have a "Recommend Content" feature for coaches), and we may want to think about that guided experience being quite different from the current flow anyway, once we get there.

**Comment by radinamatic** [comment]
> and no need for the H1 on there or under Topics.

Breadcrumbs might be optional, but H1 should not be. Every page must have a main heading.

**Comment by jamalex** [comment]
> Breadcrumbs might be optional, but H1 should not be. Every page must have a main heading.

How about `sr-only` (and hidden)? Visually, it doesn't add any value, as it's identical to the nav button right above it, and it wastes a lot of space. (Or is there an aria tag for the nav button that would allow it to act as the main heading?)

**Comment by jamalex** [comment]
Maybe this is a separate issue, but in the same vein of avoiding cruft and maximizing use of space: when there's only one type of content in the current topic, could we just hide the "Display" dropdown? It just ends up being confusing, without adding value, in that case.

**Comment by indirectlylit** [comment]
> I still actually feel it could be useful to have the topic-tree-based breadcrumbs show up on content pages instead, under Recommended, as it's useful to see where you are in the tree even when you popped in from the Recommended tile page

I disagree with this. More and more I'm thinking that the topic tree is a coach and curator-facing organizational convention. At least in coach-driven learning environments, the topic tree will often be irrelevant for learners (at least once assigned content is a thing).

**Comment by DXCanas** [comment]
Is the verdict here that we hide breadcrumbs are hidden if they're only 1 item? Are we also `sr-only`ing the header?

**Comment by khangmach** [comment]
yes hiding breadcrumbs if they're only one item. The existence of the crumb on the first level serves no navigational purpose and does not have explicit meaning (other than being a clone of the page heading) until the user gets to the second level.

**Comment by DXCanas** [comment]
- Get rid of only breadcrumb @ top level for first pass.
- Title will stay for first pass
    - more discussion around layout consistency needed re: @khangmach @indirectlylit @jtamiace

**Comment by DXCanas** [comment]
Should this be a global change or just learn?

**Comment by khangmach** [comment]
@DXCanas Global change.

**Comment by DXCanas** [comment]
For posterity, I talked about the changes made with Richard on Slack:
> Me:
Hey Richard. I'm looking into addressing the titleception bug, and I've thought of a few options to have this change happen globally, but I'm not sure which would be better.
> 1) Manually going in to every instance of breadcrumb and `v-if`ing it so that it won't even exist if the items passed in are < 1. I don't like this because it sounds annoying, and it'd be repeated anytime we want to use breadcrumbs.
> 2) Adding to core `breadcrumbs`'s validator to have it fail if the prop is < 1. I don't like this because it'll have warning messages when running the deveserver.
> 3) `v-if` the top level element inside of the component. I don't like this because all the logic in the component is still gonna happen.
you have an suggestion?

> Richard: 
I think 3 is the quickest fix
We need to revisit what we are doing in general, so I like it because it is quick and easily reversible
My sense is that we will probably reverse this change in some form, once we work out how to make the breadcrumb work better for us in combination with the other header information on the page

Looks like the approach taken is causing some issues with element references. @christianmemije mentioned that what would probaly be best is to do this inside of `learn` only, @khangmach.

If not, I could go with option 2 above, which would delegate the responsibility to the `breadcrumb` parent.

Also, @christianmemije I completely thought I had checked the console for errors and stuff. Thanks for catching that.
